app/uk/gov/hmrc/plasticpackagingtax/registration/models/genericregistration/IncorporationDetails.scala

```diff
@@ -19,20 +19,27 @@ package uk.gov.hmrc.plasticpackagingtax.registration.models.genericregistration
 import play.api.libs.functional.syntax.{unlift, _}
 import play.api.libs.json._
 
-case class IncorporationDetails(companyNumber: String, companyName: String, ctutr: String)
+case class IncorporationDetails(
+  companyNumber: String,
+  companyName: String,
+  ctutr: String,
+  companyAddress: IncorporationAddressDetails
+)
 
 object IncorporationDetails {
 
   val apiReads: Reads[IncorporationDetails] = (
     (__ \ "companyProfile" \ "companyNumber").read[String] and
       (__ \ "companyProfile" \ "companyName").read[String] and
-      (__ \ "ctutr").read[String]
+      (__ \ "ctutr").read[String] and
+      (__ \ "companyProfile" \ "unsanitisedCHROAddress").read[IncorporationAddressDetails]
```

Comment:
just a question, is this single address entry? can they send multiple addresses here?

Reply:
Single address